bolt/include/bolt/Passes/ContinuityStats.h

```diff
@@ -0,0 +1,42 @@
+//===- bolt/Passes/ContinuityStats.h - function cfg continuity analysis ---*-
+// C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Conduct function CFG continuity analysis.
```

Comment:
When I saw this PR, I am guessing that the purpose of this pass is to give an indication of profile quality.
My best guess based on this one sentence and the PR description is that the pass looks for inconsistency in profile information, where from the structure of the CFG it can be "proven" that the profile info isn't accurate.

I think it would be useful to add a description of what the pass aims to measure and report in a bit more detail.
Here, at the top of the file would be a reasonably good place to do so, I think. (Alternatively, rather than in the header, it can be done in the .cpp file)
Or is there a public document that can be pointed to that describes the high-level ideas behind this analyis?